core-war/src/main/java/org/silverpeas/web/todo/control/ToDoSessionController.java

```diff
@@ -534,11 +406,6 @@ public String initSelectionPeas() {
 
   /**
    * Retourne une Collection de UserDetail des utilisateurs selectionnés via le userPanel
-   *
-   * @param
-   * @return
-   * @throws TodoException
-   * @see
    */
   public Collection<Attendee> getUserSelected() throws TodoException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'org.silverpeas.web.todo.control.TodoException', as it cannot be thrown from method's body. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)